roo.gemspec

```diff
@@ -13,8 +13,8 @@ Gem::Specification.new do |spec|
   spec.homepage               = 'https://github.com/roo-rb/roo'
   spec.license                = 'MIT'
 
-  spec.files                  = `git ls-files -z`.split("\x0")
-  spec.files.reject! { |fn| fn.include?('test/files') }
+  spec.files                  = Dir.glob('lib/**/*').reject { |f| File.directory?(f) } + %w{CHANGELOG.md LICENSE README.md Gemfile roo.gemspec}
```

Comment:
Why exclude `directory?`
iI it is not necessary action, i would like to accept [his suggestion](https://github.com/roo-rb/roo/pull/565#issuecomment-997139205)

For reference, [rails](https://github.com/rails/rails/blob/9bdd29c8317a9cc2faa72cf2060758dd41b8c267/activemodel/activemodel.gemspec#L20) also uses this style